livebooks/elixir_ASCII_heart.livemd

```diff
@@ -55,11 +55,14 @@ The code is split into 5 parts:
 3. The function `math_power` takes two arguments, x and y, and returns a boolean, indicating whether the point (x, y) is inside the heart shape or not.
```

Comment:
Why math power?

Reply:
To make it more transferrable to future ASCII/elixir blog posts. There's a triangle waiting already :-)